.github/workflows/config/spellcheck_config.yml

```diff
@@ -152,13 +152,16 @@ matrix:
       - open: '(?s)(?P<open>\s*`{3,})\S*\s*$'
         close: '(?P=open)$'
       # Ignore multiline content fenced by \code ... \endcode
-      - open: '(?s)^\s*(\\code)'
+      - open: '(?s)\\code'
         close: '\\endcode$'
       # Ignore content between inline backticks
       - open: '(?P<open>`+)'
         close: '(?P=open)'
-      # Ignore the first lowercase word after @param, which is usually the parameter name
-      - open: '@param(?:\s[a-z]+)?'
+      # Ignore the first word after @param, which is usually the parameter name
+      - open: '@param(?:\s\w+)?'
```

Comment:
These are going to be a bit brittle. `\param<sp><tab>foo` is legal doxygen. In fact, the `\param` command has other arguments as well. `\param[out]<tab><sp><sp>egghead<tab>A defunct whiolesaler` is valid syntax.

In my changes in the other PR, I'm trying to open this up to allow people to write doxygen comments without stepping on landmines and then deciding it isn't worth making the effort.

Reply:
> In my changes in the other PR, I'm trying to open this up to allow people to write doxygen comments without stepping on landmines and then deciding it isn't worth making the effort.

During my latest merge from main (which includes your PR), I undid my change for this section (and left your change). The only change I needed to make from there was to reorder the rules because the `'[\\@]([acep])(?:\s+\S)?'` rule was pre-empting the `param` rule, and that was causing failures. Let me know if that looks reasonable to you.